bridge-sdk/eth-proof/src/eth_rpc_client/mod.rs

```diff
@@ -3,14 +3,17 @@ use reqwest::Client;
 use ::serde::Deserialize;
 use serde_json::{json, Value};
 use ethereum_types::{H256, U64};
-use crate::common::{Result, SdkError};
 
 pub mod types;
 mod serde;
 
-impl From<reqwest::Error> for SdkError {
+#[derive(thiserror::Error, Debug)]
+#[error("Ethereum RPC error: {0}")]
+pub struct EthRpcError(String);
```

Comment:
Please don't convert the `reqwest::Error` to a `string`; the information about the request error, e.g., StatusCode, can be useful for the SDK user. 
You need to create an enum with `Parse` and `reqwest::Error` errors.

Reply:
Converted to enum